src/NzbDrone.Core/Datastore/Migration/091_added_indexerstatus.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using FluentMigrator;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(91)]
+    public class added_indexerstatus : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Create.TableForModel("IndexerStatus")
+                  .WithColumn("IndexerId").AsInt32().NotNullable().Unique()
+                  .WithColumn("FirstFailure").AsDateTime().Nullable()
+                  .WithColumn("LastFailure").AsDateTime().Nullable()
+                  .WithColumn("FailureEscalation").AsInt32().NotNullable()
+                  .WithColumn("BackOffDate").AsDateTime().Nullable()
+                  .WithColumn("LastRecentSearch").AsDateTime().Nullable()
```

Comment:
It's actually MostRecent-RecentSearch. Recent = RssSync. It's not actually a Search.